Dockerfile.h2

```diff
@@ -1,10 +1,10 @@
-FROM gradle:6.2.2-jdk8 as builder
+FROM gradle:6.2.2-jdk11 as builder
 WORKDIR /app
 COPY . .
 RUN gradle dist --no-daemon
 
-FROM openjdk:8-jre-alpine
-RUN apk update && apk upgrade && apk add --no-cache bash curl wget unzip
+FROM openjdk:11-jre-slim
+RUN apt-get update && apt-get install -y bash curl wget unzip
```

Comment:
It's not an alpine anymore... I pulled the official opendjk image which is also scanned for log4shell

![image](https://user-images.githubusercontent.com/3920663/152057229-fac3f3c0-6e7c-4d0b-86cf-337ffcda850e.png)